README.md

```diff
@@ -73,6 +73,7 @@ Please read the [Contributing guide](./CONTRIBUTING.md). Thank you to all our [c
 ### Go
 
 * [Ignite CLI](https://github.com/ignite/cli) - All-in-one platform to build, launch, and maintain any crypto application on a sovereign and secured blockchain. Quickly bootstraps a new Cosmos SDK blockchain with UI and support to create new and work conveniently with existing Cosmos SDK modules.
+* [Reecepbcups/cosmos-submit-tx-client](https://github.com/Reecepbcups/cosmos-submit-tx-client) - Basic Transaction submit client in Go.
```

Comment:
This isn't something that fits here, I think. This isn't meant to be used as a library nor configurable enough to fit many use cases (e.g. https://github.com/Reecepbcups/cosmos-submit-tx-client/blob/main/helpers.go#L43-L54).
I think a tutorial should be created with such content and published in https://tutorials.cosmos.network cc @samricotta